src/main/drivers/dshot_dpwm.h

```diff
@@ -55,16 +59,19 @@ motorDevice_t *dshotPwmDevInit(const struct motorDevConfig_s *motorConfig, uint1
 #define DSHOT_DMA_BUFFER_SIZE   18 /* resolution + frame reset (2us) */
 #define PROSHOT_DMA_BUFFER_SIZE 6  /* resolution + frame reset (2us) */
 
-#define DSHOT_TELEMETRY_INPUT_LEN 32
-#define PROSHOT_TELEMETRY_INPUT_LEN 8
+#define GCR_TELEMETRY_INPUT_LEN MAX_GCR_EDGES
 
 // For H7, DMA buffer is placed in a dedicated segment for coherency management
 #if defined(STM32H7)
 #define DSHOT_DMA_BUFFER_ATTRIBUTE DMA_RAM
 #elif defined(STM32F7)
 #define DSHOT_DMA_BUFFER_ATTRIBUTE FAST_RAM_ZERO_INIT
 #else
-#define DSHOT_DMA_BUFFER_ATTRIBUTE // None
+#ifdef STM32F7
```

Comment:
This will never be true.

Reply:
Good catch - this was from the last rebase since jflyper has now fixed it